histomicstk/preprocessing/color_deconvolution/__init__.py

```diff
@@ -4,19 +4,29 @@
 """
 # make functions available at the package level using shadow imports
 # since we mostly have one function per file
+from . import stain_color_map as _stain_color_map
 from .color_convolution import color_convolution
 from .complement_stain_matrix import complement_stain_matrix
 from .sparse_color_deconvolution import sparse_color_deconvolution
 
 # must be imported after ComplementStainMatrix
 from .color_deconvolution import color_deconvolution
 
+stain_color_map = _stain_color_map.stain_color_map
+__doc__ += _stain_color_map.__doc__
```

Comment:
You can document variables in modules or classes for sphinx generated documentation using syntax like this:
```python
#: this is my description
foo
```
or 
```python
foo #: this is my description
```